tests/unit/components/form-test.js

```diff
@@ -1,253 +1,149 @@
-import {expect} from 'chai'
+import Ember from 'ember'
 const {RSVP} = Ember
+import {expect} from 'chai'
 import {describeComponent} from 'ember-mocha'
 import {afterEach, beforeEach, describe, it} from 'mocha'
-import {PropTypes} from 'ember-prop-types'
-import {validatePropTypes} from 'dummy/tests/helpers/template'
-
-describeComponent(
-  'frost-bunsen-form',
-  'Unit: Component | frost-bunsen-form',
-  {
-    unit: true
-  },
-  function () {
-    let component, onChangeSpy, onValidationSpy, sandbox
-
-    beforeEach(function () {
-      sandbox = sinon.sandbox.create()
-      onChangeSpy = sandbox.spy()
-      onValidationSpy = sandbox.spy()
-      component = this.subject({
-        bunsenModel: {
-          properties: {
-            bar: {type: 'string'},
-            baz: {type: 'number'},
-            foo: {type: 'string'}
-          },
-          required: ['foo'],
-          type: 'object'
+import {unitTest} from 'dummy/tests/helpers/template'
+
+describeComponent(...unitTest('frost-bunsen-form'), function () {
+  let component, onChangeSpy, onValidationSpy, sandbox
+
+  beforeEach(function () {
+    sandbox = sinon.sandbox.create()
+    onChangeSpy = sandbox.spy()
+    onValidationSpy = sandbox.spy()
+    component = this.subject({
+      bunsenModel: {
+        properties: {
+          bar: {type: 'string'},
+          baz: {type: 'number'},
+          foo: {type: 'string'}
         },
-        onChange: onChangeSpy,
-        onValidation: onValidationSpy
-      })
-    })
-
-    afterEach(function () {
-      sandbox.restore()
+        required: ['foo'],
+        type: 'object'
+      },
+      onChange: onChangeSpy,
+      onValidation: onValidationSpy
     })
+  })
 
-    validatePropTypes({
-      autofocus: PropTypes.bool,
-      bunsenModel: PropTypes.oneOfType([
-        PropTypes.EmberObject,
-        PropTypes.object
-      ]).isRequired,
-      bunsenView: PropTypes.oneOfType([
-        PropTypes.EmberObject,
-        PropTypes.object
-      ]),
-      disabled: PropTypes.bool,
-      onChange: PropTypes.func,
-      onValidation: PropTypes.func,
-      renderers: PropTypes.oneOfType([
-        PropTypes.EmberObject,
-        PropTypes.object
-      ]),
-      showAllErrors: PropTypes.bool,
-      validators: PropTypes.array,
-      value: PropTypes.oneOfType([
-        PropTypes.EmberObject,
-        PropTypes.null,
-        PropTypes.object
-      ])
-    })
+  afterEach(function () {
+    sandbox.restore()
+  })
 
-    it('actions.onTabChange() updates selectedTabIndex', function () {
-      [0, 1, 2].forEach((index) => {
-        component.actions.onTabChange.call(component, index)
-        expect(component.get('selectedTabIndex')).to.eql(index)
-      })
+  it('actions.onTabChange() updates selectedTabIndex', function () {
+    [0, 1, 2].forEach((index) => {
+      component.send('onTabChange', index)
```

Comment:
This is one piece I wanted to make sure I called out. If you want to call an action on a component, you don't need to use
 ```js
component.actions.actionName.call(component, param)
``` 
you can use 
```js
component.send('actionName', param)
```